src/helm/manifests/ralph-env-cm.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
This should be a distributed template instead, with all configurations overridable by defining the corresponding Value.